src/index.js

```diff
@@ -125,16 +132,44 @@ class JanusAdapter {
 
   async addOccupant(occupantId) {
     var subscriber = await this.createSubscriber(occupantId);
+
+    this.occupants[occupantId] = true;
```

Comment:
Why have 3 seperate datastrucures here? Seems like you just want a map from ocupantId -> subscriber

Reply:
naf has the occupant map as `{ [string:occupantId]: boolean] }`, but it would still be truthy... I'd have to check to see if that breaks anything.